server-spi-private/src/main/java/org/keycloak/broker/provider/AbstractIdentityProvider.java

```diff
@@ -33,7 +33,9 @@
 import java.nio.charset.StandardCharsets;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
+import java.util.Arrays;
```

Comment:
Are changes to this class unrelated?

Reply:
Good catch, the imports can be removed now. I fixed that.